tests/cupyx_tests/fallback_mode_tests/test_notifications.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import unittest
+import contextlib
+from io import StringIO
```

Comment:
Reorder them alphabetical order.

Reply:
Importing non-module symbols is prohibited.

Reply:
Done